lib/dns/verifier.js

```diff
@@ -165,13 +178,21 @@ async function verifyWithDS(key) {
   return sets;
 }
 
-async function filterRRs(rrs, qtype) {
+function filterRRs(rrs, qtype) {
   return rrs.filter(r => {
     return r.type == qtype;
   });
 }
 
-async function getRRset(rrs, name, qtype) {
+function filterNSEC(rrs, qtype) {
```

Comment:
qtype seems to be unused? Why not make this a general purpose function that takes a list of acceptable types?